exampleCourse/questions/demo/annotated/LectureVelocity/1-Introduction/question.html

```diff
@@ -17,9 +17,9 @@
 
 <p> How is the average rate of change of a function connected to its graph? Take a look at the graph below:</p>
 
-<p>
-<pl-figure file-name="figure1.png" type="dynamic" width="400"></pl-figure>
-</p>
+<div class="p-2">
+  <pl-figure file-name="figure1.png" type="dynamic" width="400"></pl-figure>
```

Comment:
What do you think about baking a bottom margin into all `pl-figure` and `pl-drawing` elements? Alternatively, how bad do things look without the spacing at all?

Is there a reason you went for `p-2` over `mb-3`, which mirrors the bottom margin on `<p>` and friends?

Reply:
> What do you think about baking a bottom margin into all pl-figure and pl-drawing elements? 

I think it's a good idea -- as long as we aren't worried about adding twice the spacing for existing instructor elements.

> Is there a reason you went for p-2 over mb-3, which mirrors the bottom margin on <p> and friends?

Nope -- I can easily swap this out.

Reply:
Changed in [f13e235](https://github.com/PrairieLearn/PrairieLearn/pull/11898/commits/f13e23552396a6b5f15ce1978156d13e829ca7ca).